packages/@aws-cdk/aws-glue-alpha/lib/storage-parameter.ts

```diff
@@ -0,0 +1,323 @@
+/**
+ * The compression type used by `StorageParameters.COMPRESSION_TYPE`.
```

Comment:
What I was getting at with my other questions: How can I check this is the complete list?

Reply:
This is not a complete list, and has been specified within the JSDOC of the attribute that uses this enum (`storageParameters`). However this may not be apparent to some users. Should I make this apparent within the README as well?

Reply:
Gotcha. Yes please include these note into the enum as well.

Reply:
I've added a `@see`